recipes/hexl/all/test_package/example.cpp

```diff
@@ -0,0 +1,133 @@
+#include <cstdint>
```

Comment:
It could be reduced to a single small example. The idea is testing the package only.

Reply:
Done